### ISSUE TITLE
Fix Request: false positive in AvoidUnconditionalBuiltLogStrings

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#430]
Message is logged and returned, so a flase positive:
```
import lombok.extern.slf4j.Slf4j;

@Slf4j
class Foo {
    String bar() {
        StringBuilder messageBuilder = new StringBuilder();
        messageBuilder.append("-"); // <-- false positive
    
        log.info("message: {}" , messageBuilder);

        return messageBuilder.toString();
    }
}
```
